plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/core/CoreResources.properties

```
@@ -366,6 +366,8 @@ pref_page_ui_general_combo_language_tip = Application language - used for user i
 pref_page_ui_general_group_browser = Client Web Engine
 pref_page_ui_general_combo_browser = Web Engine
 pref_page_ui_general_combo_browser_tip = Used for displaying web contents inside the application
+pref_page_ui_general_check_browser_auth = Open auth links in embedded browser
+pref_page_ui_general_check_browser_auth_tip = Will redirect SSO and other auth requests to be shown in the embedded window
```

The sentence is malformed. It could be `Will redirect SSO and other auth requests to the embedded window` or `Will make SSO and other auth requests to be shown in the embedded window`
```suggestion
pref_page_ui_general_check_browser_auth_tip = Will redirect SSO and other auth requests to be shown in the embedded window
```